### PR TITLE
doctest the example in MessageEncryptor

### PR DESCRIPTION
**Pull request by CrowdHailer**

### CODE REVIEW

File: lib/plug/crypto/message_encryptor.ex [hunk 1]
```diff
@@ -11,17 +11,17 @@ defmodule Plug.Crypto.MessageEncryptor do
...
-      sign_secret = KeyGenerator.generate(secret_key_base, encrypted_signed_cookie_salt)
-
-      data = "José"
-      encrypted = MessageEncryptor.encrypt(data, secret, sign_secret)
-      decrypted = MessageEncryptor.decrypt(encrypted, secret, sign_secret)
-      decrypted # => {:ok, "José"}
+      iex> secret_key_base = "072d1e0157c008193fe48a670cce031faa4e..."
+      ...> encrypted_cookie_salt = "encrypted cookie"
+      ...> encrypted_signed_cookie_salt = "signed encrypted cookie"
+      #
```
Comment (CrowdHailer) [inline lib/plug/crypto/message_encryptor.ex]: Is there a way to have blank lines in a doctest other than adding this empty comment?

Comment (josevalim) [reply lib/plug/crypto/message_encryptor.ex]: Can you do `...>` and leave it empty?

Comment (CrowdHailer) [reply lib/plug/crypto/message_encryptor.ex]: Turns out yes :-)

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**